llvm/lib/SYCLLowerIR/SYCLPropagateAspectsUsage.cpp

```diff
@@ -377,6 +406,20 @@ buildFunctionsToAspectsMap(Module &M, TypeToAspectsMapTy &TypesWithAspects) {
 
 } // anonymous namespace
 
+SPAUOptions SYCLPropagateAspectsUsagePass::parseOpts(StringRef Params) {
+  SPAUOptions Result;
+  if (!Params.empty()) {
+    size_t Pos = 0, SeparatorPos = 0;
+    while (SeparatorPos != Params.npos) {
+      SeparatorPos = Params.find(",", Pos);
+      auto Target = Params.substr(Pos, SeparatorPos - Pos);
+      Result.Targets.push_back(Target);
+      Pos = SeparatorPos + 1;
+    }
+  }
```

Comment:
```suggestion
  Params.split(Result.Targets, ',', /*MaxSplit=*/-1, /*KeepEmpty=*/false);
```
See [`StringRef::split`](https://llvm.org/doxygen/classllvm_1_1StringRef.html#af0284e4c41c0e09c0bc4767bc77a899d)

Reply:
Oh, I didn't notice that there is such a useful variant of `split`, thanks! [7043cdc](https://github.com/intel/llvm/pull/7115/commits/7043cdc3454840ab6d7ef5b5f953d8062205b5a1)